mpxe/integration_tests/test_mcp3427.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import time
+
+from mpxe.integration_tests import int_test
+from mpxe.sims.mcp3427 import Mcp3427
+
+
+class TestMcp3427(int_test.IntTest):
+    def setUp(self):
+        super().setUp()
+        self.mcp3427 = self.manager.start('smoke_mcp3427', Mcp3427())
+
+    def test_mcp3427(self):
+        time.sleep(1)
```

Comment:
I mentioned it above, but you should have a update_mcp3427_reading() function in the sim, which you can call with 20 as the value here and then the assert_store_value line below should work the same